README.md

```diff
@@ -55,24 +60,37 @@ Passing an object of class NOT extending SerializabilityTrait as a message may c
 test0.scala:6: error: Make sure this type is itself annotated, or extends a type annotated with  @org.virtuslab.ash.annotation.SerializabilityTrait.
   sealed trait Command extends MySerializable
                ^
+```
+
+The compiler plugin only checks the classes in the sbt modules where `AkkaSerializationHelperPlugin` is explicitly enabled.
+It may happen that your base trait (like `MySerializable` in the example) lives in an sbt module like `core` where you do **not** want to enable the plugin
+(e.g. for compilation performance reasons).
+You still need to reference `org.virtuslab.ash.SerializabilityTrait` to annotate `MySerializable`, however.
+In order to have access to the `SerializabilityTrait` annotation without enabling the entire suite of compiler plugins,
+add `AkkaSerializationHelperPlugin.annotation` to `libraryDependencies`:
 
+```scala
+import org.virtuslab.ash.AkkaSerializationHelperPlugin
+
+lazy val core = (project in file("core"))
+  .settings(libraryDependencies += AkkaSerializationHelperPlugin.annotation)
 ```
 
 ### 2. Dump Persistence Schema
 
 A mix of a compiler plugin and an sbt task for dumping schema
 of [akka-persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html#example-and-core-api) to a
-file. It can be used for detecting accidental changes of events and states with simple `diff`.
+file. It can be used for detecting accidental changes of events (journal) and states (snapshots) with a simple `diff`.
 
-To dump persistence schema, run:
+To dump persistence schema for each sbt module where `AkkaSerializationHelperPlugin` is enabled, run:
 
 ```shell
 sbt ashDumpPersistenceSchema
 ```
 
 Default file is `<sbt-module>/target/<sbt-module-name>-dump-persistence-schema-<version>.yaml` but it can be changed using sbt keys:
```

Comment:
To be reconsidered (maybe just single `target/dump-persistence-schema-<version>.yaml`)

Reply:
It will be after #131